modules/rhcos-add-extensions.adoc

```diff
@@ -14,6 +14,8 @@ Currently, the following extensions are available:
 
 * **kerberos**: Adding the `kerberos` extension provides a mechanism that allows both users and machines to identify themselves to the network to receive defined, limited access to the areas and services that an administrator has configured. See link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/system-level_authentication_guide/using_kerberos[Using Kerberos] for details, including how to set up a Kerberos client and mount a Kerberized NFS share.
 
+* **sysstat**: Adding the `sysstat` extension provides additional performance monitoring for {product-title} nodes, including the system activity reporter (`sar`) command for collecting and reporting information. See link:https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/html-single/monitoring_and_managing_system_status_and_performance/index#proc_providing-feedback-on-red-hat-documentation_monitoring-and-managing-system-status-and-performance[Monitoring and managing system status and performance] for details.
```

Comment:
Does adding the `sysstat` extension provide additional performance monitoring for nodes AND the system activity reporter (`sar`), 

OR

Does adding the `sysstat` extension provide additional performance monitoring for nodes and the ABILITY to use the `sar` command? 

Just a thought. 

Looks good overall.

Reply:
it's both 🤪

Reply:
i put emphasis on `sar` because that is the tool from the package that customers are most interested in, according to support